spec/lib/axe/configuration_spec.rb

```diff
@@ -26,7 +26,11 @@ module Axe
     it { is_expected.to respond_to :wait_interval }
     it { is_expected.to respond_to :wait_interval= }
 
-    its(:jslib) { is_expected.to start_with("/*! aXe") }
+		# We have removed comments from `axe.min.js`, so excluding this test
+		# Hence cannot do start_with("/*! aXe"), instead do a function we know should exist check
+    its(:jslib) { 
+			is_expected.to include("axe.run=function(")
+		}
```

Comment:
Weird indentation. tabs vs spaces?

Reply:
Github rendering I believe, looks pristine on IDE. Have seen this issue before on Github.

![image](https://user-images.githubusercontent.com/20978252/44743336-1f641a00-aafa-11e8-91ea-fbf0f813e5cc.png)

Reply:
It's definitely in the source. The arrows in the screenshot are tabs. Your ide must just be configured to have a tab width that makes them appear like two spaces. (Or otherwise line them up correctly.)

![image](https://user-images.githubusercontent.com/119972/44743837-91743900-aad1-11e8-970f-530feeb95f7d.png)

There's also trailing whitespace in there.